addons/product/tests/common.py

```diff
@@ -14,7 +14,11 @@ class ProductCommon(
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-
+        decimal_precision = cls.env['decimal.precision'].search([
```

Comment:
I think it's useless.
If people change the kind of base data, of course things will break.

For me, it's like if someone were to change some access rights to models on their db and complain that tests would break.
Or if he changes the name of the report in config of Accounting.
Same kind of base data that people might change in a real db and that we don't re-force in a test

But if you really want to enforce it then whatever.

Reply:
@gawa-odoo Ok you got the point. You can remove that part